### PR TITLE
chore(python-docs): update documentation for python client

### PR DESCRIPTION
**Pull request by joybytes**
## Summary
- additional fields were added to get notification endpoint [as part of this PR](https://github.com/alphagov/notifications-api/pull/4105) in our public API
- this PR updates the documentation details for our python client

### Related task

[Add GET notification cost data to API clients and API client docs](https://trello.com/c/5VZrBNpU/845-3-add-get-notification-cost-data-to-api-clients-and-api-client-docs)

### CODE REVIEW

File: source/documentation/client_docs/_python.md [hunk 1]
```diff
@@ -617,30 +617,41 @@ If the request to the client is successful, the client will return a `dict`:
...
+    "line_2": "ADDRESS LINE 2",  # required string for letter
+    "line_3": "ADDRESS LINE 3",  # optional string for letter
+    "line_4": "ADDRESS LINE 4",  # optional string for letter
+    "line_5": "ADDRESS LINE 5",  # optional string for letter
+    "line_6": "ADDRESS LINE 6",  # optional string for letter
+    "line_7": "ADDRESS LINE 7",  # optional string for letter
+    "type": "sms / letter / email",  # required string
+    "status": "sending / delivered / permanent-failure / temporary-failure / technical-failure",  # required string
+    "template": {
+        "Version": 1,
```
Comment (joybytes) [inline source/documentation/client_docs/_python.md:634]: @CrystalPea we seem to be missing on our docs if the field `Version` is required or optional. Could you let me know so I can update it here?

Comment (CrystalPea) [reply source/documentation/client_docs/_python.md:634]: Hmm I think we'd always have a template version, so it is required.

**Review by CrystalPea — APPROVED** [review]
That looks good to me 👍🏼

File: source/documentation/client_docs/_python.md (hunk 1 above)
Comment (joybytes) [reply source/documentation/client_docs/_python.md]: I included the missing fields you added on your PR too
https://github.com/alphagov/notifications-tech-docs/pull/233/commits/f79973b4ab7f3620e562703618e712adce0aacbc

**Review by CrystalPea — APPROVED** [review]

**Pull request merged**